docs/config.rst

```diff
@@ -149,6 +149,17 @@ Those configuration options are documented below:
             sampleRate: 0.5 // send 50% of events, drop the other half
         }
 
+.. describe:: sanitizeKeys
+
+    An array of strings representing keys that should be scrubbed from the payload sent to Sentry.
```

Comment:
(worth being explicit here that this doesn't scrub or match on string values or query encoded data)

Reply:
also worth mentioning that sentry itself can do do server side sanitizing and this is different

Reply:
basically i think people are really gonna want to know what guarantees they get from this feature so it's worth writing a longer explanation